### ISSUE TITLE
Review all models for key uniqueness

### ISSUE THREAD

**Issue by OscarPDR** [OscarPDR/labman_ud#127]

**Comment by porduna** [comment]
* New events are generated with a unique slug. But we have to migrate old events.
* There is no uniqueness for the table PublicationAuthor (author, publication). We manually check that when adding authors so it's not a big deal, but they should ensure uniqueness.

**Comment by OscarPDR** [comment]
Yeah, I think that "relation" tables such as PublicationAuthor (link between Publication and Person) should ensure the uniqueness through all the ForeignKeys. There are lots of models formed this way, so a huge revision is welcomed. We need to solve the migrations issue first, though...
